code/game/objects/items/storage/book.dm

```diff
@@ -244,6 +244,18 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 			REMOVE_TRAIT(sword, TRAIT_NODROP, HAND_REPLACEMENT_TRAIT) //in case the "sword" is a possessed dummy
 			user.visible_message("<span class='notice'>[user] exorcises [sword]!</span>", \
 								"<span class='notice'>You successfully exorcise [sword]!</span>")
+	else if(istype(A, /obj/item/ai_module/core/full/drone))
+		var/obj/item/ai_module/core/full/drone/unholylawset = A //this lawset was hand-crafted by satan and you can't convince me otherwise
+		to_chat(user, "<span class='notice'>You begin to exorcise [unholylawset]...</span>")
+		playsound(src,'sound/hallucinations/veryfar_noise.ogg',40,TRUE)
+		if(do_after(user, 40, target = unholylawset))
+			playsound(src, 'sound/voice/borg_deathsound.ogg', 200, TRUE, TRUE) //sadly, drones don't have a unique deathsound, so the generic silicon death noise will have to do
+			new /obj/item/ai_module/core/full/damaged(get_turf(unholylawset))
+			user.visible_message("<span class='notice'>[user] purifies [unholylawset]!</span>")
+			qdel(unholylawset)
```

Comment:
not a fan of this in terms of design or implementation.

If you are adding things to chaplain it should be modular per sect 😏

Reply:
I'm not adding it to the chaplain, I'm adding it to the bible.

This functions identically to both possessed blade purification and soulstone purification, which can both be performed by non-chaplains.

Furthermore, aren't sects currently not liked by oranges?

Reply:
misread but it makes no sense for a bible to convert a board.

oranges doesnt like sects as an avenue for magic powers which im quite conscious of, and i think even if it isnt tied to a sect a bible converting boards is pushing more specifically into the realm he takes issue with.

Reply:
> misread but it makes no sense for a bible to convert a board.

The joke is that the board is unholy and needs to be exorcised.

Also, do we even currently HAVE a system for converting items based on your sect?

Reply:
the joke isnt funny and no not currently, i look forward to your contribution though 😏

Reply:
humor is subjective